temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -125,4 +125,41 @@ service CloudService {
             get: "/api/v1/cloud/regions/{region}",
         };
     }
+
+    // Get all known API keys
+    rpc GetAPIKeys (GetAPIKeysRequest) returns (GetAPIKeysResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/cloud/api-keys",
+        };
+    }
+
+    // Get an API key
+    rpc GetAPIKey (GetAPIKeyRequest) returns (GetAPIKeyResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/cloud/api-keys/{api_key_id}",
```

Comment:
```suggestion
            get: "/api/v1/cloud/api-keys/{key_id}",
```

Reply:
The param name should be the same as the field name.

Reply:
done